test/local_creds_test.go

```diff
@@ -162,18 +160,15 @@ func spoofDialer(addr net.Addr) func(target string, t time.Duration) (net.Conn,
 	}
 }
 
-func testLocalCredsE2EFail(dopts []grpc.DialOption) error {
+func testLocalCredsE2EFail(t *testing.T, dopts []grpc.DialOption) error {
 	ss := &stubserver.StubServer{
 		EmptyCallF: func(context.Context, *testpb.Empty) (*testpb.Empty, error) {
```

Comment:
same. Listener needs to be assigned here?

Reply:
Done